feature/cilium/feature.yaml

```diff
@@ -0,0 +1,5 @@
+icon: ""
```

Comment:
Any idea for an icon? Maybe 𐄳? 😄

Reply:
![grafik](https://github.com/dodevops/cloudcontrol/assets/13124453/7903fb9d-5649-48b1-ade7-53f471f143f1)
 https://cilium.io/favicon-32x32.png